src/Microsoft.DotNet.Wpf/src/PresentationUI/Themes/Generic.xaml

```diff
@@ -8449,7 +8449,7 @@ style should be placed here.
                                               ToolTip="Previous Page"
                                               Command="NavigationCommands.PreviousPage"
                                               IsEnabled="{TemplateBinding CanGoToPreviousPage}"
-                                              Focusable="False">
+                                              Focusable="True">

```

Comment:
Annotate .NET Framework's file and see if there was ever an explicit reason these were not focusable.  You can also search some of the older bug databases for something.

It doesn't feel like these should have been configured this way, but sometimes there are good reasons.